Modules/Relationships/test/Relationships.Domain.Tests/Tests/Aggregates/Relationships/DecomposeRelationshipTests.cs

```diff
@@ -90,4 +107,21 @@ public void Identity_to_can_only_decompose_once()
         // Assert
         acting.Should().Throw<DomainException>().WithError("error.platform.validation.relationshipRequest.relationshipAlreadyDecomposed");
     }
+
+    [Fact]
+    public void Two_identities_can_enter_into_a_new_relationship_again_after_decomposing()
```

Comment:
This test should be part of the CreateRelationshipTests

Reply:
Test moved to correct place.